pyquidax/wrappers/accounts.py

```diff
@@ -5,12 +5,33 @@
 
 
 class Account(BaseAPIWrapper):
+    """
+    A class that represent Account and has a BaseAPIWrapper as the object
```

Comment:
I'm glad you attempted the docstrings. It shows you read about docstrings. We should have a meet before you work on docstrings. This type of documentation is bad because it's repeating the obvious